core/domain/collection_domain_test.py

```diff
@@ -31,7 +31,7 @@
 
 SAMPLE_YAML_CONTENT = ("""category: A category
 language_code: en
-next_skill_id: 2
```

Comment:
Oh no. This means we need a migration actually, since the stuff in the datastore is already in the wrong format. Let me help you with this. I'll send in a PR to migrate, and once that's done, we'll get that merged into develop and then you can merge it into your PR. But don't change the YAML stuff here.

Reply:
updated.